src/build/__tests__/build-test.js

```diff
@@ -149,4 +168,16 @@ describe('build()', () => {
       Timestamp: { serialize },
     }, undefined, false)).forEach((type, i) => expectTypesEqual(type, [Record, Timestamp][i]));
   });
+
+  it('should allow directives configuration using __schema', () => {
+    const config = { __schema: { directives: [customDirective] } };
+    const target = build(schemaSource + querySourceWithDirective, config, undefined, false);
```

Comment:
Didn't get this point... Should I remove the last two arguments (`undefined` and `false`)? Or just `false' only?